cfme/tests/automate/custom_button/test_custom_menu.py

```diff
@@ -54,4 +79,11 @@ def test_custom_menu_display():
     Bugzilla:
         1678151
     """
-    pass
+    appliance.update_advanced_settings(ADVANCE_SETTING_CUSTOM_MENU)
+    appliance.reboot()
+    view = navigate_to(appliance.server, "LoggedIn")
+
+    for menu in ["RedHat", "ManageIQ"]:
+        view.navigation.select(menu)
+        assert view.navigation.currently_selected == [menu]
+        assert "id={}".format(menu.lower()) in appliance.browser.selenium.current_url
```

Comment:
**Question** Is there no cleanup after the test completes? I would like to see the menu being removed